site/docs/es/hosting/fly.md

```diff
@@ -0,0 +1,433 @@
+
+```json{19}
+// package.json
+{
+  "name": "grammy",
+  "version": "1.0.0",
+  "description": "grammy",
+  "main": "app.js",
+  "author": "itsmeMario",
+  "license": "MIT",
+  "dependencies": {
+    "express": "^4.18.1",
+    "grammy": "^1.11.0"
+  },
+  "devDependencies": {
+    "@types/express": "^4.17.14",
+    "@types/node": "^18.7.18",
+    "typescript": "^4.8.3"
+  },
+  "engines": {
+    "node": "16.14.0"
+  }
+}
+```
+
+</CodeGroupItem>
+</CodeGroup>
+
+4. Edita `app` dentro del archivo `fly.toml`.
+   La ruta `./app.ts` (o `./app.js` para Node.js) en el ejemplo de abajo se refiere al directorio del archivo principal.
+   Puedes modificarlos para que coincidan con el directorio de tu proyecto.
+   Si estás usando webhooks, asegúrate de que el puerto es el mismo que el de tu [configuración](#webhooks) (`8000`).
+
+<CodeGroup>
+<CodeGroupItem title="Deno (Webhooks)" Active>
+
+```toml{7,11,12}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "run --allow-net ./app.ts"
+
+[[services]]
+  http_checks = []
+  internal_port = 8000
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+  [services.concurrency]
+    hard_limit = 25
+    soft_limit = 20
+    type = "connections"
+
+  [[services.ports]]
+    force_https = true
+    handlers = ["http"]
+    port = 80
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = 443
+
+  [[services.tcp_checks]]
+    grace_period = "1s"
+    interval = "15s"
+    restart_limit = 0
+    timeout = "2s"
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Deno (Long polling)" Active>
+
+```toml{7}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "run --allow-net ./app.ts"
+
+# Simply omitting the whole [[services]] section 
+# since we are not listening to HTTP
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Node.js (Webhooks)" Active>
+
+```toml{7,11,18,19}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "node ./build/app.js"
+
+# Adjust the NODE_ENV environment to suppress the warning
+[build.args]
+  NODE_ENV = "production"
+
+[build]
+  builder = "heroku/buildpacks:20"
+
+[[services]]
+  http_checks = []
+  internal_port = 8000
+  processes = ["app"]
+  protocol = "tcp"
+  script_checks = []
+  [services.concurrency]
+    hard_limit = 25
+    soft_limit = 20
+    type = "connections"
+
+  [[services.ports]]
+    force_https = true
+    handlers = ["http"]
+    port = 80
+
+  [[services.ports]]
+    handlers = ["tls", "http"]
+    port = 443
+
+  [[services.tcp_checks]]
+    grace_period = "1s"
+    interval = "15s"
+    restart_limit = 0
+    timeout = "2s"
+```
+
+</CodeGroupItem>
+<CodeGroupItem title="Node.js (Long polling)" Active>
+
+```toml{7,11,22,23}
+# fly.toml
+app = "grammy"
+kill_signal = "SIGINT"
+kill_timeout = 5
+
+[processes]
+  app = "node ./build/app.js"
+
+# Adjust the NODE_ENV environment to suppress the warning
```

Comment:
This should be translated

Reply:
Will be fixed in https://github.com/grammyjs/website/pull/558

Thanks